worlds/stardew_valley/test/stability/TestStability.py

```diff
@@ -54,3 +53,6 @@ def test_all_locations_and_items_are_the_same_between_two_generations(self):
             # We check that the actual rule has the same order to make sure it is evaluated in the same order,
             #  so performance tests are repeatable as much as possible.
             self.assertEqual(rule_a, rule_b, f"Location rule of {location_a} at index {i} is different between both executions. Seed={seed}")
+
+        for key, value in result_a["slot_data"].items():
```

Comment:
I would personally put each of these 4 `for`s in a subtest, as I find it more clean in the test results, but I won't die on this hill. Can ignore.

Reply:
I will indeed ignore, thanks tho.